src/main/java/com/arpnetworking/tsdcore/sinks/HttpPostSinkActor.java

```diff
@@ -100,6 +100,10 @@ public HttpPostSinkActor(
         } else {
             _spreadingDelayMillis = new Random().nextInt((int) spreadPeriod.toMillis());
         }
+        LOGGER.info()
+                .setMessage("Http post sink actor spread period")
+                .addData("spreadPeriodMillis", _spreadingDelayMillis)
```

Comment:
Can you some of the other configuration parameters here and change the message to "Starting HTTP post sink actor"